src/Drivers/StorageDriver.php

```diff
@@ -2,49 +2,60 @@
 
 namespace RonasIT\Support\AutoDoc\Drivers;
 
-use Illuminate\Support\Facades\Storage;
 use Illuminate\Contracts\Filesystem\FileNotFoundException;
+use Illuminate\Support\Facades\Storage;
+use RonasIT\Support\AutoDoc\Exceptions\MissedProductionFilePathException;
 use RonasIT\Support\AutoDoc\Interfaces\SwaggerDriverInterface;
 
 class StorageDriver implements SwaggerDriverInterface
 {
     protected $disk;
-    protected $filePath;
-
-    protected static $data;
+    protected $prodFilePath;
+    protected $tempFilePath;
 
     public function __construct()
     {
-        $this->disk = config('auto-doc.drivers.storage.disk');
-        $this->filePath = config('auto-doc.drivers.storage.production_path');
+        $this->disk = Storage::disk(config('auto-doc.drivers.storage.disk'));
+        $this->prodFilePath = config('auto-doc.drivers.storage.production_path');
+        $this->tempFilePath = 'temp_documentation.json';
+
+        if (empty($this->prodFilePath)) {
+            throw new MissedProductionFilePathException();
+        }
     }
 
-    public function saveTmpData($tempData)
+    public function saveTmpData($data)
     {
-        self::$data = $tempData;
+        $this->disk->put($this->tempFilePath, json_encode($data));
```

Comment:
tmp data should be stored in the local file, but storage disc may be set as remote storage which may cause a slow testing process. Please use file_put_contetn as well here